src/libnvme-mi.map

```diff
@@ -18,6 +18,8 @@ LIBNVME_MI_1_1 {
 		nvme_mi_admin_set_features;
 		nvme_mi_admin_ns_mgmt;
 		nvme_mi_admin_ns_attach;
+		nvme_mi_admin_fw_download;
+		nvme_mi_admin_fw_commit;
```

Comment:
I think these should go into LIBNVME_MI_1_2

Reply:
ah, we should probably move others into `1_2` too then. There have been a few new definitions since 1.1 was released.

I'll do that as a first change, which this would then apply on top of.

Reply:
That's now up at https://github.com/linux-nvme/libnvme/pull/484, I'll rebase this change.